src/shared/chat/message-body.js

```diff
@@ -37,6 +37,7 @@ export default class MessageBody extends CustomElement {
 
     onImgClick (ev) { // eslint-disable-line class-methods-use-this
         ev.preventDefault();
+        api.modal.remove('converse-image-modal');
```

Comment:
Thanks for this PR. I need to look into the issue in a bit more detail. I don't think this is the right approach, ideally the modal should be removed when closed (not re-opened) or it should be re-used without having to close it first.

I'll take a look once I have some time available.

Reply:
I'm not able to reproduce this bug. I tried multiple times in different chats and every time I can open the image modal multiple times.

Can you reproduce the bug without your fix (and on the latest `master` branch) and explain to me how I can also reproduce it?

Reply:
Hi , @jcbrand , i cannot reproduce the problem in not customised Conversejs either, [my test](https://www.loom.com/share/d38beb2e01ac4772b18148c02b1f65e4?sid=25c9b3b6-8c14-43d2-8142-56adcb06a9ea). 

Please kindly reject this PR. 

To provide background info on this error: we are working on Converse customisation and decided to contribute back fixes that we made in our implementation. In our implementation the problem did exist and fixed the way as it is submitted with this PR. Or customisation changes is more mobile device oriented UI layout.  Since it is hard to find the problem root cause we decided just to re-init modal window every time it is invoked. I do not see drawbacks in this approach i might be unaware of some conversejs working details. Is the modal window reusing justified in this context ? IMO it adds complexity to the code that impacts development experience and cost. I.e. no benefits in this particular case.